app/server/models/user.py

```diff
@@ -506,6 +514,51 @@ def pin_failed_attempts(self):
     def user_details(self):
         "{} {} {}".format(self.first_name, self.last_name, self.phone)
 
+    def get_most_relevant_transfer_usage(self):
+        '''Finds the transfer usage/business categories there are most relevant for the user
+        based on the last number of send and completed credit transfers supplemented with the
+        defaul business categories
+        :return: list of most relevant transfer usage objects for the usage
+        """
+        '''
+
+        # I spent way too much time failing to figure out how to write
+        # this query using sqlalchemy if someone could point met how to translate
+        # this query to sqlalchemy please do. 
+        # Can it maybe use the paginate_query function?
+        sql = text('''
```

Comment:
Like the comment says if. If someone has a suggest of how to convert this to an sqlalchemy query

Reply:
i won't be of any help on sqlalchemy, but could we have done the rest as part of the sql query? `group_by u.business_usage_id` + `count`?

Reply:
joins are surprisingly annoying in SQLAlchemy. TBH leaving this in SQL is probably fine for the time being IMO

Reply:
`WHERE sender_user_id = 4 AND c.transfer_status = 'COMPLETE'`
Why is there sender_user_id = 4? Seems like an error?

Reply:
@RubenGeo @wangfowen @enjeyw

Reply:
You are totally right! I a pull to fix this: https://github.com/teamsempo/SempoBlockchain/pull/52